SequenceAnalysis/src/org/labkey/sequenceanalysis/query/SequenceAnalysisCustomizer.java

```diff
@@ -99,21 +100,21 @@ private void customizeSharedCols(AbstractTableInfo ti)
     private enum COL_ENUM
     {
         refNtSequence(Integer.class, PageFlowUtil.set("sequenceid", "ref_nt_sequence")){
-            public void customizeColumn(BaseColumnInfo col, AbstractTableInfo ti)
```

Comment:
this probably is only coming b/c of the branch mismatch, right? can you rebase off discvr-21.3?

Reply:
Just updated in commit "reset SequenceAnalysisCustomizer to 21.3 version"